www/pages/about/community.md

```diff
@@ -1,3 +1,10 @@
+---
+label: 'community'
+menu: side
+title: 'Community'
```

Comment:
Is there a need to add custom `title` or `label` to all these pages?  I think Greenwood's graph / resolvers should be able to provide defaults for all these already, right?

Reply:
Yes, the front-matter title is for the individual unique meta title for head page.

Reply:
Yeah, I was mistaken about that.  Seems then like a good opportunity for an issue to add default title inference.